k2/python/k2/fsa.py

```diff
@@ -13,6 +13,10 @@
 from . import fsa_properties
 from .autograd_utils import phantom_set_scores_to
 
+
+import k2.utils
```

Comment:
I would suggest
```python
from .utils import to_dot
```

I think `import k2.utils` requires that we add `the/path/to/k2` to `PYTHONPATH`.